build-tools/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -172,6 +189,13 @@ public void beforeStart() {
                     node.setting("xpack.security.transport.ssl.keystore.path", "transport.keystore");
                     node.setting("xpack.security.transport.ssl.certificate_authorities", "transport.ca");
                 }
+
+                if (apmServerEnabled) {
+                    ApmServerBuildService apmServerBuildService = getMockApmServer().get();
+                    node.setting("telemetry.metrics.enabled", "true");
```

Comment:
should we also start tracing by default?
tracing is noisy and still can be enabled with -Dtests.es.tracing.apm.enabled=true

Reply:
Let's start with metrics.